cooperator/models/subscription_request.py

```diff
@@ -344,7 +344,7 @@ def _compute_subscription_amount(self):
         required=True,
         readonly=True,
         states={"draft": [("readonly", False)]},
-        default=lambda self: datetime.strftime(datetime.now(), "%Y-%m-%d"),
+        default=lambda self: datetime.strftime(datetime.now(), "%Y-%m-%d"),  # fixme
```

Comment:
```suggestion
        default=lambda: date.today(),
```
(needs to import `date` from `datetime`)